### PR TITLE
Save file to disk

### PR DESCRIPTION
**Pull request by marghidanu**
Adds the ability to save the output as files on disk.

```
make DIRECTORY=PrimePerl/solution_1 FORMATTER=json OUTPUT_FILE=$PWD/output.json
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -10,8 +10,9 @@ all: benchmark
 
 .PHONY: benchmark
 benchmark: check-env
-	@REALPATH=$$(cd "$${DIRECTORY}" && pwd); cd tools/; \
-	npm ci --silent && npm start --silent -- benchmark -d "$${REALPATH}" -f "$(FORMATTER)"
+	@REALPATH=$$(cd "$${DIRECTORY}" && pwd); \
+	ARGS=""; [ ! -z $${OUTPUT_FILE} ] && ARGS+="-o $${OUTPUT_FILE}"; \
+	cd ./tools; npm ci --silent && npm start --silent -- benchmark -d "$${REALPATH}" -f "$(FORMATTER)" $${ARGS}
```
Comment (rbergen) [inline Makefile]: This being the first optional argument it doesn't matter much yet, but if/when another optional argument is added, consistently managing the whitespaces between arguments would become something to consider. To prepare for that scenario, one approach you could consider is this:

```bash
	ARGS=""; [ ! -z $${OUTPUT_FILE} ] && ARGS+=" -o $${OUTPUT_FILE}"; \
	cd ./tools; npm ci --silent && npm start --silent -- benchmark -d "$${REALPATH}" -f "$(FORMATTER)"$${ARGS}
```

Comment (marghidanu) [reply Makefile]: I think it’s better if I make ARGS a bash array, that way I get the spaces for free :)

Comment (marghidanu) [reply Makefile]: @rbergen Done. I think it's better with a bash array.

Comment (rbergen) [reply Makefile]: I agree. :)

**Review by rbergen — APPROVED** [review]
LGTM

**Pull request merged**